src/Akeneo/CouplingDetector/Configuration/Configuration.php

```diff
@@ -16,23 +17,24 @@ class Configuration
     /** @var RuleInterface[] */
     private $rules;
 
+    /** @var ExclusionInterface[] */
+    private $exclusions;
+
     /** @var Finder */
     private $finder;
 
     /**
      * Configuration constructor.
      *
-     * @param RuleInterface[] $rules
-     * @param Finder          $finder
+     * @param Finder               $finder
+     * @param RuleInterface[]      $rules
+     * @param ExclusionInterface[] $exclusions
      */
-    public function __construct(array $rules, Finder $finder = null)
+    public function __construct(Finder $finder, array $rules, array $exclusions = [])
     {
-        if (null === $finder) {
```

Comment:
only cause i changed the order of args but yes it's cool to have a default finder, in other hand i ask myself if the fact to define it from the conf file does not make easier to understand that it's customizable